check.py

```diff
@@ -413,6 +413,7 @@ def fix(x):
     print s
     f = open('a.js', 'w')
     f.write(open(os.path.join(options.binaryen_bin, 'binaryen.js')).read())
+    f.write('var Binaryen = module.exports;\n')
```

Comment:
Sorry to go back around on this, but this troubles me. I'd like binaryen.js to "just work" in the easiest way for people. That means

1. If you include it on your web page or bundle it alongside your JS code, you can just call `Binaryen.*`, without needing that line of boilerplate.
2. If you use it in a unified module loader context, it works in a consistent way with that approach.

So I would like to avoid adding this in `check.py`, and having it in binaryen.js seems natural for 1. Is it impossible to achieve both 1 and 2 at the same time?